test/third_party/install.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
```

Comment:
My thinking was that we would add any subsequent `third_party` stuff to this file. If you like, I could make this file simply invoke other shell scripts and give this one a more descriptive name.